<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionSetsWindow</class>
 <widget class="QMainWindow" name="AdditionSetsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>183</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>241</width>
    <height>183</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>241</width>
    <height>183</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Дополнительная настройка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SetBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Block1RBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Блок 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Block2RBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Блок 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Для калибровки при расположении &quot;соосно&quot; и &quot;соосно развёрнуто&quot; необходимо выбрать тип блока.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пожалуйста выберите тип блока:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
